docker/docker-compose.yml

```diff
@@ -33,24 +35,21 @@ services:
       - GIRDER_MONGO_URI=mongodb://mongo:27017/girder
       - GIRDER_ADMIN_USER=${GIRDER_ADMIN_USER}
       - GIRDER_ADMIN_PASS=${GIRDER_ADMIN_PASS}
+      - CELERY_BROKER_URL=amqp://guest:guest@rabbit/
+      - BROKER_CONNECTION_TIMEOUT=2
 
   girder_worker:
+    runtime: 'nvidia'
+    build:
+      context: ../
+      dockerfile: docker/girder_worker.Dockerfile
     volumes:
-      - ./server:/home/viame_girder
       - pipelines:/home/VIAME/build/install/configs/pipelines
```

Comment:
Do we have to coordinate this with the change in `girder_worker.Dockerfile`? As now these files should be at `/opt/noaa/viame/configs/pipelines/` in the girder worker image, right?

Reply:
This is `./server`, a bind-mount of the source code.  I don't think the source code should be mounted in the production docker-compose.  It is, however, added in the dev compose.

I don't think this has anything to do with pipelines -- could you make sure that's true?

Reply:
These lines https://github.com/VIAME/VIAME-Web/blob/001031fe99b5799ab4da7a441c1f4a65c778cd19/docker/docker-compose.yml#L24 https://github.com/VIAME/VIAME-Web/blob/001031fe99b5799ab4da7a441c1f4a65c778cd19/docker/docker-compose.yml#L45

This is what allows girder to expose the list of runnable pipelines through an API endpoint. I just tested this locally and it doesn't expose them properly, as it's trying to look at `/home/VIAME/build/install/configs/pipelines`. It seems like the place it should be pointing at is `/opt/noaa/viame/configs/pipelines/`, however I don't know if there's a difference there between all pipelines, and ones that are enabled from the install. There was before (which is why I pulled from the `install` sub-directory).  It doesn't seem like there's an `install` folder in that new path. I think the image this is using has them all enabled, which I think means that the list of files at `/opt/noaa/viame/configs/pipelines/` should be good. @mattdawkins could you weigh in?

Reply:
You're right, I'm dumb.

Reply:
43ef72f seems to fix this issue.